dags/experiments_live.py

```diff
@@ -45,6 +45,17 @@
         "moz-fx-data-shared-prod.telemetry_derived.experiment_unenrollment_overall_v1"
     ]
 
+    export_monitoring_data = gke_command(
+        task_id="export_monitoring_data",
+        command=[
+            "python",
+            "script/experiments/export_experiment_monitoring_data.py",
+            "--date", "{{ ds }}",
+            "--datasets"
+        ] + datasets,
+        docker_image=docker_image
+    )
+
```

Comment:
Taking of the pressure of Airflow a little by running the export script in one slot for all the queries, instead of one slot for each query.